src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -26,7 +30,7 @@
 				/>
 			</div>
 			<div class="flex-1">
-				<h6 class="text-sm font-bold">{$i18n.dapps.text.featured}</h6>
+				<h6 class="text-sm font-bold text-[#0BA5EC]">{$i18n.dapps.text.featured}</h6>
```

Comment:
We don't have any hardcoded color at the moment, maybe worth keeping that approach and creating a new color?

Reply:
Yes i can introduce another blue, I'll name it info-blue. TBH I think we should match the colors of the figma (I did this with https://github.com/dfinity/oisy-wallet/pull/2897 which is waiting to be reviewed).

Reply:
In case you're wondering why info blue, this matches the variable names in figma 
![Uploading image.png…]()